spec.emu

```diff
@@ -4,11 +4,799 @@
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %Int16Array%
+              </td>
+              <td>
+                `Int16Array`
+              </td>
+              <td>
+                The Int16Array constructor (<emu-xref href="#sec-typedarray-objects"></emu-xref>)
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %Int32Array%
+              </td>
+              <td>
+                `Int32Array`
+              </td>
+              <td>
+                The Int32Array constructor (<emu-xref href="#sec-typedarray-objects"></emu-xref>)
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %isFinite%
+              </td>
+              <td>
+                `isFinite`
+              </td>
+              <td>
+                The `isFinite` function (<emu-xref href="#sec-isfinite-number"></emu-xref>)
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %isNaN%
+              </td>
+              <td>
+                `isNaN`
+              </td>
+              <td>
+                The `isNaN` function (<emu-xref href="#sec-isnan-number"></emu-xref>)
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %IteratorPrototype%
+              </td>
+              <td>
+              </td>
+              <td>
+                An object that all standard built-in iterator objects indirectly inherit from
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %JSON%
+              </td>
+              <td>
+                `JSON`
+              </td>
+              <td>
+                The `JSON` object (<emu-xref href="#sec-json-object"></emu-xref>)
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %Map%
+              </td>
+              <td>
+                `Map`
+              </td>
+              <td>
+                The Map constructor (<emu-xref href="#sec-map-constructor"></emu-xref>)
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %MapIteratorPrototype%
+              </td>
+              <td>
+              </td>
+              <td>
+                The prototype of Map iterator objects (<emu-xref href="#sec-map-iterator-objects"></emu-xref>)
+              </td>
+            </tr>
+            <tr>
+              <td>
+                %Math%
+              </td>
+              <td>
+                `Math`
+              </td>
+              <td>
+                The `Math` object (<emu-xref href="#sec-math-object"></emu-xref>)
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <ins>%ModuleSource%</ins>
+              </td>
+              <td>
+              </td>
```

Comment:
```suggestion
              <td>
                `ModuleSource`
              </td>
```

I thought we were going to expose `ModuleSource` on the global object.

Reply:
It's _not_ listed here as exposed, we should list it if it is (in the currently empty column).